src/allmydata/test/test_util.py

```diff
@@ -13,16 +13,19 @@
 import six
 import os, time, sys
 import yaml
+import json
 
 from twisted.trial import unittest
 
 from allmydata.util import idlib, mathutil
 from allmydata.util import fileutil
+from allmydata.util import jsonbytes
```

Comment:
`jsonbytes` would seem a good candidate for a separate PR to aid digestion, if desired.